README.md

```diff
@@ -1 +1,15 @@
-# pyconuk-2018-k8s
\ No newline at end of file
+# pyconuk-2018-k8s
+
+# Learning Objectives
+* What is Kuberrnetes?
+* Get familiar with Kubernetes vocabulary.
+* Install and run Kubernetes on `localhost`.
+* Interact with the Kubernetes API using `kubectl`.
+* Use the Python client for Kubernetes to create deployments.
```

Comment:
Not deployments necessary I think,just using the python client to do things with kubernetes (the example will probably be pods/jobs)